src/master-playlist-controller.js

```diff
@@ -363,9 +247,9 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     let label = null;
 
     // if no label was passed in we are switching to the currently enabled audio
-    for (let i = 0; i < this.hlsHandler.tech_.audioTracks().length; i++) {
-      if (this.hlsHandler.tech_.audioTracks()[i].enabled) {
-        label = this.hlsHandler.tech_.audioTracks()[i].label;
+    for (let i = 0; i < this.tech_.audioTracks().length; i++) {
```

Comment:
I know it's not really in scope, but it would be so much prettier with with find() and a polyfill for IE.

Otherwise, again out of scope, but it may read easier if we create a variable for this.tech_.audioTracks()